packages/grafana-ui/src/components/IconButton/IconButton.tsx

```diff
@@ -121,7 +121,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme2, size, variant: IconButton
     button: css({
       zIndex: 0,
       position: 'relative',
-      margin: theme.spacing(0, 0.5, 0, 0),
+      margin: `0 ${theme.spacing.x0_5} 0 0`,
```

Comment:
I would still recommend/prefer the spacing function as it creates easier to read and understand code. 

```ts
button: css({
      margin: theme.spacing(0,5, 1)
})
```

is soo much clearer and easier to read than 

```ts
button: css({
      margin: `${theme.spacing.x0_5} ${theme.spacing.x1}`
})
```

Reply:
I agree that it is simpler, but we are _hoping_ that with the layout components there is less of a need to set margins and this becomes even easier to do than with css 😄

Reply:
@JoaoSilvaGrafana I really hope the Layout component take grid units as numbers not forcing using these tokens as that would be so unreliable and messy to have to do `spacing={[theme.spacing.x0_5, theme.spacing_x1]}` that is going to make every frontend dev cry

Reply:
Yes it will definitely be numbers! `spacing={[0.5, 1]}`
We expect the tokens to be used directly only when someone wants to override some margin/padding, even though we will recommend using a Layout component for most use cases.